indico/modules/events/controllers/management.py

```diff
@@ -39,3 +39,30 @@ def _process(self):
             tpl = get_template_module('events/management/_reference_list.html')
             return jsonify_data(html=tpl.render_event_references_list(self.event_new.references))
         return jsonify_form(form)
+
+
+class RHManageLocation(RHConferenceModifBase):
+
```

Comment:
Not important: I think we don't leave an empty line between the class declaration and the body.